x-pack/plugins/ml/server/routes/schemas/fields_service_schema.ts

```diff
@@ -29,4 +29,6 @@ export const getTimeFieldRangeSchema = schema.object({
   timeFieldName: schema.maybe(schema.string()),
   /** Query to match documents in the index(es). */
   query: schema.maybe(schema.any()),
+  /** Additional search options. */
+  indicesOptions: schema.maybe(schema.any()),
```

Comment:
Since this `indicesOptions` is used in a lot of end points, should we define the schema for it?

Reply:
https://github.com/elastic/kibana/pull/91830/commits/76a4479e5a402ec0593f4a8e89f0559514295192